DDPFF/run/colorpcd.py

```diff
@@ -0,0 +1,25 @@
+import sys
+import numpy as np
+import open3d as o3d
+
+
+path = 'input/result.ply'
+pcd = o3d.io.read_point_cloud(path)
+
+planes = []                
+with open('output/planes.txt', 'r') as f:
+    for line in f:
+        planes.append([int(x) for x in line.split()])
```

Comment:
here you can create np.array for each plane like `np.asarray([int(x) for x in line.split()])` to use in future

Reply:
Done